### PR TITLE
New Url Input

### LINKED ISSUE #157: {{boostrap-inputs/url}}
As a creative/technical user I need documentation and samples for {{boostrap-inputs/url}} so that I can reference and implement a bootstrap-input control for url.

### PR DESCRIPTION
**Pull request by jweakley**
Pull Request Description:
New `url` input. Fixes #157

New Pull Request Checklist:
- [x] Reviewed [contributing guidelines](CONTRIBUTING.md).
- [x] [good commit messages](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html)
- [x] Github issue added, if it exists.
- [x] Added an entry to the [Changelog](CHANGELOG.md).
- [ ] All tests passing and new component tests include [ember-a11y-testing](https://github.com/ember-a11y/ember-a11y-testing) to test accessibility.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ All notable changes to this project will be documented in this file.
   * [#142](https://github.com/wildland/ember-bootstrap-controls/issues/142): Added contribution guidelines and documentations. Also added blueprints. ([@jweakley][])
   * [#145](https://github.com/wildland/ember-bootstrap-controls/issues/145): Added new email type input. ([@jweakley][])
   * [#146](https://github.com/wildland/ember-bootstrap-controls/issues/146): Added new password type input. ([@Nathan-O][])
+  * [#156](https://github.com/wildland/ember-bootstrap-controls/issues/156): Added new url type input. ([@jweakley][])
```
Comment (harokevin) [inline CHANGELOG.md]: Is #156 ({{boostrap-inputs/time}}) the correct issue number? Shouldn't it be #157 ({{boostrap-inputs/url}})?

File: blueprints/bootstrap-component/files/app/components/__name__.js [hunk 2]
```diff
@@ -0,0 +1 @@
+export { default } from 'ember-bootstrap-controls/components/<%= dasherizedModuleName %>';
```
Comment (harokevin) [inline blueprints/bootstrap-component/files/app/components/__name__.js]: Is this real?

File: blueprints/freestyle-component/files/addon/templates/components/freestyle/__name__.hbs [hunk 3]
```diff
@@ -5,12 +5,12 @@
     as |collection|
   }}
     {{#collection.variant key='Basic Usage'}}
-      {{#freestyle-usage 'basic-<%= dasherizedModuleName %>' name='Basic <%= classifiedPackageName %>'}}
+      {{#freestyle-usage 'basic-<%= entity.freestyleName %>' name='Basic <%= classifiedPackageName %>'}}
```
Comment (harokevin) [inline blueprints/freestyle-component/files/addon/templates/components/freestyle/__name__.hbs]: what are dis?

File: blueprints/freestyle-component/files/addon/templates/components/freestyle/__name__.hbs [hunk 4]
```diff
@@ -5,12 +5,12 @@
...
         {{<%= dasherizedModuleName %>}}
       {{/freestyle-usage}}
     {{/collection.variant}}
     {{#collection.variant key='Advanced Usage'}}
-      {{#freestyle-usage 'advanced-<%= dasherizedModuleName %>' name='Advanced <%= classifiedPackageName %>'}}
+      {{#freestyle-usage 'advanced-<%= entity.freestyleName %>' name='Advanced <%= classifiedPackageName %>'}}
```
Comment (harokevin) [inline blueprints/freestyle-component/files/addon/templates/components/freestyle/__name__.hbs]: stahp

Comment (jweakley) [reply blueprints/freestyle-component/files/addon/templates/components/freestyle/__name__.hbs]: Yes. That is an ember blueprint file. They are used by the generators.

File: blueprints/freestyle-component/files/addon/templates/components/freestyle/__name__.hbs (hunk 3 above)
Comment (jweakley) [reply blueprints/freestyle-component/files/addon/templates/components/freestyle/__name__.hbs]: Yes. That is an ember blueprint file. They are used by the generators.

File: blueprints/bootstrap-component/files/app/components/__name__.js (hunk 2 above)
Comment (jweakley) [reply blueprints/bootstrap-component/files/app/components/__name__.js]: Yes. That is an ember blueprint file. They are used by the generators.

**Pull request merged**